lib/generators/monologue/views_generator.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails/generators/base'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.